admin/config.default.sh

```diff
@@ -15,6 +15,13 @@ JSON_DUMP_GROUP=musicbrainz
 JSON_DUMP_DIR_MODE=755
 JSON_DUMP_FILE_MODE=644
 
+# Same, but for search indexes dumps.
+SEARCH_INDEXES_DUMP_DIR=/home/musicbrainz/search-indexes-dumps
```

Comment:
nitpick: search-indexes-dumps sounds strange because indexes and dumps are both plural. "search-index-dumps" makes more sense to my ear (for referring to a collection of dumps). though "search indexes dump" (as used in admin/RunSearchIndexesDump for example) also makes sense for referring to the dump as a whole (or the singular dump process)

Reply:
To clarify (just in case):
Search index refer to a specific collection in Solr, for example: `release-group`.
Each dump saves all of the 16 search indexes.
The above directory contains 2 to 3 dumps. (The 2 latest plus the current one.)

Should I still rename it to prevent double plural? (4 occurrences in this PR, 9 in docker server configs)

Reply:
Well, if it were up to me I'd rename it just 'cause it sounds ungrammatical, but I don't have a technical reason to force it. "Search index dumps" could mean any number of indexes per dump, so it's fine in that respect.

Reply:
Renamed in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/e10fb6bbfdf06687cb119e9769110fa1b1ddc1c6

Please let me know if these changes are alright before I do the same in docker server configs and mb solr cloud.

Reply:
Yes, that looks alright to me 👍

Reply:
Thanks, changed docker server configs PR, no change needed in mb solr cloud.